HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -7,12 +7,26 @@ class HVAC
   AirSourceCoolRatedIWB = 67.0 # degF, Rated indoor wetbulb for air-source systems, cooling
   CrankcaseHeaterTemp = 50.0 # degF
 
-  def self.apply_air_source_hvac_systems(model, cooling_system, heating_system,
+  def self.apply_air_source_hvac_systems(model, runner, cooling_system, heating_system,
                                          sequential_cool_load_fracs, sequential_heat_load_fracs,
                                          weather_max_drybulb, weather_min_drybulb,
-                                         control_zone, hvac_unavailable_periods)
+                                         control_zone, hvac_unavailable_periods, schedules_file, hpxml_bldg)
     is_heatpump = false
+    if not schedules_file.nil?
+      max_pow_ratio_sch = schedules_file.create_schedule_file(model, col_name: SchedulesFile::ColumnMaximumPowerRatio, schedule_type_limits_name: Constants.ScheduleTypeLimitsFraction)
+      # Not allowed with unit multiplier for now
+      if not max_pow_ratio_sch.nil?
+        fail 'Maximum power ratio schedules of variable speed hvac systems are only supported if NumberofUnits is 1.' if hpxml_bldg.building_construction.number_of_units > 1
+      end
```

Comment:
Good check, but please be consistent with other building technologies (see [here](https://github.com/NREL/OpenStudio-HPXML/blob/4f60b2e19cb413803022cbafef31d28e083f820e/HPXMLtoOpenStudio/resources/hvac.rb#L262-L266) and [here](https://github.com/NREL/OpenStudio-HPXML/blob/4f60b2e19cb413803022cbafef31d28e083f820e/HPXMLtoOpenStudio/resources/hvac.rb#L777-L781)). So: "NumberofUnits greater than 1 is not supported for variable speed HVAC systems with maximum power ratio schedules." And add the FUTURE comment and link to github issue like we do for the others.